queues/nats.md

```diff
@@ -76,6 +76,8 @@ jobs:
 
 `stream` - stream name.
 
+> Notice: To avoid duplicate message consumption in NATS-driven pipelines, use unique streams for each pipeline. Sharing streams can lead to repeated processing of the same messages.
```

Comment:
This is the obvious notice. But ok, more docs is good. Notice should be done in the standard way. {% %} search for this syntax.